kolibri_explore_plugin/assets/src/views/AssessmentWrapper/index.vue

```diff
@@ -224,13 +225,11 @@ oriented data synchronization.
               } else if (this.correct === 0 && this.hintWasTaken) {
                 // not correct and hint
                 return 'hint';
-              } else {
-                // not correct and no hint
-                return 'wrong';
```

Comment:
Can you explain in the commit message what are the changes in AssessmentWrapper about? Is this related to adding content cards to the search result?

Reply:
That should be an automatic linter change, I didn't remember updating that, so it should be a change that's done during my commits after running the `yarn run lint-frontend:format` and I added it. It's not related to this task and we can remove these changes from the PR.

Reply:
I've removed this change.